### DISCUSSION TITLE
RNDIS Host?

### DISCUSSION THREAD

**Discussion by HansNelisse-SH** [hathach/tinyusb#1752] [Q&A]
I notice TinyUsb has RNDIS in the Device stack. Would it be possible to include RNDIS to the Host stack?

We are looking for a solution which can do that. We need soemething to connect to a 3rd party RNDIS USB device.
In the past we have used LUFA on an AT90USB1287 chip, and got a working prototype. But that chip is now out of stock (and its specs were probably not sufficient for our application). 
We are now considering the RP2040 chip, and TinyUsb looks like an interesting way to go.

**Comment by hathach** [comment]
it is possible but no one is currently working on rndis host.

**Reply by AndyW999** [reply]
I would like this too ;)
